queries.sql

```diff
@@ -15,3 +15,52 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] @Fene-87 great work on your query, however, you forgot to add a query before and after the commit to verify the changes were made and persisted after the commit.

Reply:
Thanks @danmainah I have implemented the changes